src/main/java/com/superdevs/HealthOMeter/dto/AuthoritiesUsersDto.java

```diff
@@ -0,0 +1,41 @@
+package com.superdevs.HealthOMeter.dto;
+
+public class AuthoritiesUsersDto {
+
+    private Long id;
+    private Long user_id;
+    private Long authority_id;
+
+    public AuthoritiesUsersDto() {
+    }
+
+    public AuthoritiesUsersDto(Long id, Long user_id, Long authority_id) {
+        this.id = id;
```

Comment:
DTO czyli data  transfer object, nie ma odzwierciedlenia w bazie danych.. nie widzę sensu nadawania ID

Reply:
Fixing it right now, great suggestion, make sense.